innopoints/views/account.py

```diff
@@ -512,7 +513,7 @@ def reclaim_innopoints():
             log.exception(err)
             abort(400, {'message': 'Data integrity violated.'})
 
-    cur.execute('DELETE FROM User WHERE email=?', (request.json['email'],))
+    cur.execute('DELETE FROM User WHERE email=?;', (user['email'],))
```

Comment:
Why the semicolons? As far as I remember, they are not necessary. But I don't mind them

Reply:
They are not, but I still like to include them for some reason

Reply:
Probably stems from using pgAdmin, where the last statement doesn't need a semicolon, but I tend to keep them as I sometimes need to add other statements after it